README.md

```diff
@@ -15,10 +21,14 @@ export CLOUDSDK_PYTHON_SITEPACKAGES=1;
 
 
 ## Renew Load-Balancers self-managed certifications in a project
-Overview:
+### Overview:
 This script rotates load balancers SSL certificate when the certificate's remaining cert-life is less then a threshold. For example, if remaining cert-life threshold is set to  50%, then a 30 days cert will be renewed 15 days before expiration. If remaining cert life threshold is set to 90%, then a 24hrs cert will be renewed 2.4hrs before certification expiration.
 The script assumes all certificates are issued from a pre-configured subordinate CA named 'private-ca-subordinate-name'.
 
+The script is designed to run as a [cron job ](https://www.hostgator.com/help/article/what-are-cron-jobs), the certificate(s) installed on the resources listed in the YAML configuration file will be queried for certificate life-span and renewed if needed. The frequency in which this script executes depends on the certificates' life-span, for example if the certification are issued for 24 hours and expected to be renewed 6 hours before expiration, then the script should be executed every 4 hours with the following crontab schedule expression: ```0 */4 * * *```
+
+
+
 ## Architectural overview:
 
 ![Architectural overview](./PrivateCA-AutomationOverview.png)
```

Comment:
Can you clarify the section below on Auditing and Logging? It's unclear to me whether this script will write audit logs or the CA service is supposed to write those logs.

Reply:
Changed the text, i will need to wait until the documentation are public to add link(s) to what gets logged.